test/components/tbTableComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { useTbTable } from '../../src/useTbTable';
+import columns from '../fixtures/columns';
+import localData from '../fixtures/localData';
+import { TubularComponent } from './tubularComponent';
+
+export const TbTableComponent = () => {
+    const { state, api } = useTbTable(columns, localData);
+    return (
+        <>
+            <div>
+                <button onClick={() => api.goToPage(state.page + 1)}>Go to next page</button>
+                <button onClick={() => api.goToPage(state.page - 1)}>Go to previous page</button>
+                <button onClick={() => api.sortColumn('CustomerName')}>Sort by Customer Name</button>
+            </div>
+            <table>
+                <thead>
+                    <tr role="rowheader">
+                        {state.columns
+                            .filter(col => col.visible)
+                            .map(col => {
+                                return <th key={col.name}>{col.label}</th>;
+                            })}
+                    </tr>
+                </thead>
+                <tbody>
+                    {state.data.map((row, index) => {
+                        return (
+                            <tr key={index}>
+                                {state.columns
+                                    .filter(col => col.visible)
+                                    .map(col => {
+                                        return (
+                                            <td role="cell" key={col.name}>
+                                                {row[col.name]}
+                                            </td>
+                                        );
+                                    })}
```

Comment:
You don't need to use return in this lambda.

```suggestion
                                    .map(col => (
                                            <td role="cell" key={col.name}>
                                                {row[col.name]}
                                            </td>
                                        ))}
```

Reply:
Removing...